public/css/style.css

```diff
@@ -1,30 +1,103 @@
-* { margin: 0; padding: 0; box-sizing: border-box; }
-body { font: 13px Helvetica, Arial; }
-form { 
-	background: #000;
-	padding: 3px;
-	position: fixed;
-	bottom: 0;
-	width: 100%;
+* { outline:0; border: 0; margin: 0; padding: 0; font-family: 'Ubuntu Mono', monospace; }
+
+body { 
+	color: #1c1c1c;
+	background-image: linear-gradient(to bottom, #c8ddf4 0%, #e7f0fd 100%); 
 }
-form input {
-	border: 0;
-	padding: 10px;
-	width: 90%;
-	margin-right: .5%;
+
+.container {
+	display: grid;
+	grid-template-areas: "header header header header"
+						 ". sidebar content.";
```

Comment:
Fix the spacing here.
Tip: Sublime text has an "Indentation: Convert to Spaces" command.